src/routes/paths.ts

```diff
@@ -53,3 +53,25 @@ export const RELEASEPLAN_EDIT_PATH = RELEASEPLAN_PATH.extend(
 );
 
 export const RELEASEPLAN_CREATE_PATH = RELEASEPLAN_PATH.extend(`create`);
+
+// Integration test paths
+
+export const INTEGRATION_TEST_LIST_PATH = APPLICATION_DETAILS_PATH.extend('integrationtests');
+
+export const INTEGRATION_TEST_DETAILS_PATH = INTEGRATION_TEST_LIST_PATH.extend(
+  `:${RouterParams.integrationTestName}`,
+);
+
+export const INTEGRATION_TEST_ADD_PATH = INTEGRATION_TEST_LIST_PATH.extend('add');
+
+export const INTEGRATION_TEST_EDIT_PATH = INTEGRATION_TEST_DETAILS_PATH.extend('edit');
+
+// Pipeline paths
+
+export const PIPELINE_RUNS_LIST_PATH = APPLICATION_DETAILS_PATH.extend('pipelineruns');
```

Comment:
wouldn't you need activity path for pipeline run in application details?

Reply:
<img width="1421" alt="Screenshot 2025-03-04 at 8 05 30 PM" src="https://github.com/user-attachments/assets/c5affe78-6b3b-4177-bc55-8d3f82012003" />

All PLR-TaskRun paths I've put up in the TaskRun PR -> https://github.com/konflux-ci/konflux-ui/pull/137/files#diff-26212c7e58b310fe67e20a7cd969dce535cc31c1f843f741c71641dddc7320dd